### PR TITLE
Slope and some modularisation

### PR DESCRIPTION
**Pull request by pierromond**
+ Modularize deltaSlope and deltaStuddedTyres and deltaTemperature
+ add slope : // In the case of a bi-directional traffic flow, it is necessary to split the flow into two components and correct half for uphill and half for downhill.

### CODE REVIEW

**Comment by pierromond** [comment]
next step, add slope in wps : see #133

**Comment by nicolas-f** [comment]
So, an optional field on roads for separating bidirectional roads and unidirectional ones ? Which one is default ?

**Comment by pierromond** [comment]
I set unidirectional (and in the direct way) as default...but... I don't know what could be the best choise. The user should use rsparamterscnossos.setWay() to change the way.

**Comment by nicolas-f** [comment]
Not requesting update but you can bound values like that also:
```java
speed = Math.max(50, Math.min(90, speed));
```

File: noisemodelling-jdbc/src/main/java/org/noise_planet/noisemodelling/jdbc/LDENPropagationProcessData.java [hunk 1]
```diff
@@ -115,6 +115,7 @@ public void addSource(Long pk, Geometry geom, SpatialResultSet rs) throws SQLExc
         double pmStud = 0;
         double junctionDistance = 100; // no acceleration of deceleration changes with dist >= 100
         int junctionType = 2;
+        int way = 1;
```
Comment (nicolas-f) [inline noisemodelling-jdbc/src/main/java/org/noise_planet/noisemodelling/jdbc/LDENPropagationProcessData.java:118]: not 3 ?

**Comment by pierromond** [comment]
Update, I decided to also implement in WPS. Issue #133 resolved !

**Pull request merged**